android/app/src/main/res/xml/network_security_config.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<network-security-config>
+      <base-config>
+           <trust-anchors>
+                <certificates src="system" />
+                <certificates src="user" />
+           </trust-anchors>
```

Comment:
I'd like to have a comment here with a sentence or two explaining what this is doing and why, and also one for the `domain-config` element. Should link to the request https://github.com/zulip/zulip-mobile/issues/3312 , and also to the reference docs for understanding this file's semantics:
https://developer.android.com/training/articles/security-config#FileFormat

I'll probably write those comments up in the course of merging this. But if you can make a go at writing a draft, that would help me out and would be appreciated.

Reply:
I've added a number of comments to address your concern. Please let me know if it needs any further adjustment.